### ISSUE TITLE
Changes in DropShadow View [a11y] [BLOCKED on discussion]

### ISSUE THREAD

**Issue by Arjupta** [oppia/oppia-android#2531]
**Describe the bug**
 Drop shadow View of TabBar is causing the checkbox directly below it to have an unacceptable contrast ratio and hence Accessibility checks fail here.

**To Reproduce**
Steps to reproduce the behaviour:
1. Start the Accessibility Scanner App on your device
2. Open the Oppia Android App and start Fraction Exploration
3. Switch to the Practice Tab
4. Keep the device in landscape mode and scroll the contents upwards till the checkboxes start coming directly below it.
3. Scan the screenshots with the Accessibility Scanner
4. You will see the tests that are not passed by that screen

**Expected behaviour**
All the tests must be passed by scanning the screenshots for both Tablet and Mobile Devices in every orientation.

**Screenshots**
|||
|:-:|:-:|
|![Screenshot_1611207456](https://user-images.githubusercontent.com/64064110/105284920-64141280-5bd9-11eb-903c-5540f06d72e5.png)|![Screenshot_1611207443](https://user-images.githubusercontent.com/64064110/105285026-958cde00-5bd9-11eb-91b5-7bd9b5060367.png)|

**Device**
- Pixel - 3 Emulator
- SDK version - 29

**Additional context**
The Screenshots are descriptive enough for understanding the issues but still, if there is a need of clarification then you can comment here. Multiple Pull Requests can also be made for solving these issues.

Reference - [https://github.com/oppia/oppia-android/wiki/Working-on-UI#accessibility-scanner](https://github.com/oppia/oppia-android/wiki/Working-on-UI#accessibility-scanner)

**Comment by rt4914** [comment]
This issue can be closed as although it is appearing as an error but thats under very specific circumstance which won't happen on screen reader and also this does pass on mobile devices.
